src/data/javascript-quiz.ts

```diff
@@ -1,24 +1,4 @@
 const javascriptQuiz = [
-   {
-    Question: "What does the 'debugger' statement in JavaScript do?",
-    Answer: "It acts as a breakpoint in a program",
-    Distractor1: "It will debug errors in the current statement",
-    Distractor2: "It will debug all the errors in the program at runtime",
-    Distractor3: "All of the above",
-    Explanation:
-      "The 'debugger' statement is used to setup the breakup points at probable points of errors in the code snippet, using the debugger window.",
-    Link: "https://www.freecodecamp.org/news/debugging-javascript-for-beginners-5d4ac15dd1cd/#"
-  },
-  {
-    Question: "What kind of programming can JavaScript be used for?",
-    Answer: "Both Client and Server side programming",
-    Distractor1: "Server side only",
-    Distractor2: "Client side only",
-    Distractor3: "None of these options",
-    Explanation:
-      "JavaScript is a programming language which can be used on the client and server side.",
-    Link: "https://developer.mozilla.org/en-US/docs/Learn/Server-side/First_steps/Introduction#are_server-side_and_client-side_programming_the_same"
```

Comment:
Why are these questions deleted? Please do not remove questions unless a maintainer has approved those changes.

Reply:
It was mistakenly done due to a branch change.
Changes have been reverted.